third_party/ZoKrates/zokrates_pest_ast/src/lib.rs

```diff
@@ -11,18 +11,19 @@ pub use ast::{
     Access, Arguments, ArrayAccess, ArrayInitializerExpression, ArrayType, AssertionStatement,
     Assignee, AssigneeAccess, BasicOrStructType, BasicType, BinaryExpression, BinaryOperator,
     BooleanLiteralExpression, BooleanType, CallAccess, ConstantDefinition, ConstantGenericValue,
-    Curve, DecimalLiteralExpression, DecimalNumber, DecimalSuffix, DefinitionStatement, EOI,
-    ExplicitGenerics, Expression, FieldSuffix, FieldType, File, FromExpression, FromImportDirective,
-    FunctionDefinition, HexLiteralExpression, HexNumberExpression, IdentifierExpression,
-    ImportDirective, ImportSource, ImportSymbol, InlineArrayExpression, InlineStructExpression,
-    InlineStructMember, IterationStatement, LiteralExpression, MainImportDirective, MemberAccess,
-    NegOperator, NotOperator, Parameter, PosOperator, PostfixExpression, Pragma, PrivateVisibility,
-    PrivateNumber, PublicVisibility, Range, RangeOrExpression, ReturnStatement, Span, Spread,
-    SpreadOrExpression, Statement, StructDefinition, StructField, StructType, SymbolDeclaration,
-    TernaryExpression, ToExpression, Type, TypedIdentifier, TypedIdentifierOrAssignee,
-    U8NumberExpression, U16NumberExpression, U32NumberExpression, U64NumberExpression, U8Suffix,
-    U16Suffix, U32Suffix, U64Suffix, U8Type, U16Type, U32Type, U64Type, UnaryExpression,
-    UnaryOperator, Underscore, Visibility,
+    Curve, DecimalLiteralExpression, DecimalNumber, DecimalSuffix, DefinitionStatement,
+    ExplicitGenerics, Expression, FieldSuffix, FieldType, File, FromExpression,
+    FromImportDirective, FunctionDefinition, HexLiteralExpression, HexNumberExpression,
+    IdentifierExpression, ImportDirective, ImportSource, ImportSymbol, InlineArrayExpression,
+    InlineStructExpression, InlineStructMember, IterationStatement, LiteralExpression,
+    MainImportDirective, MemberAccess, NegOperator, NotOperator, Parameter, PosOperator,
+    PostfixExpression, Pragma, PrivateNumber, PrivateVisibility, PublicVisibility, Range,
+    RangeOrExpression, ReturnStatement, Span, Spread, SpreadOrExpression, Statement,
+    StructDefinition, StructField, StructType, SymbolDeclaration, TernaryExpression, ToExpression,
+    Type, TypedIdentifier, TypedIdentifierOrAssignee, U16NumberExpression, U16Suffix, U16Type,
+    U32NumberExpression, U32Suffix, U32Type, U64NumberExpression, U64Suffix, U64Type,
+    U8NumberExpression, U8Suffix, U8Type, UnaryExpression, UnaryOperator, Underscore, Visibility,
+    EOI,
```

Comment:
Huh?

Reply:
rustfmt? should probably be reverted.

Reply:
yep, this is from running `cargo fmt --all`. is there a way to make the formater ignore this section of code?

Reply:
Oh, interesting. I don't care whether we format the third_party crates or not. Thoughts, @kwantam ?